examples/asr/speech_multitask/speech_to_text_aed_chunked_infer.py

```diff
@@ -0,0 +1,326 @@
+from tqdm import tqdm
+
+from nemo.collections.asr.data.audio_to_text_lhotse_prompted import canary_prompt
+from nemo.collections.asr.models import EncDecMultiTaskModel
+from nemo.collections.asr.parts.submodules.multitask_decoding import MultiTaskDecodingConfig
+from nemo.collections.asr.parts.utils import rnnt_utils
+from nemo.collections.asr.parts.utils.eval_utils import cal_write_wer
+from nemo.collections.asr.parts.utils.streaming_utils import FrameBatchASR
+from nemo.collections.asr.parts.utils.transcribe_utils import (
+    compute_output_filename,
+    get_buffered_pred_feat,
+    setup_model,
+    wrap_transcription,
+    write_transcription,
+)
+from nemo.collections.common.parts.preprocessing.manifest import get_full_path
+from nemo.core.config import hydra_runner
+from nemo.utils import logging
+
+can_gpu = torch.cuda.is_available()
+
+
+@dataclass
+class TranscriptionConfig:
+    # Required configs
+    model_path: Optional[str] = None  # Path to a .nemo file
+    pretrained_name: Optional[str] = None  # Name of a pretrained model
+    audio_dir: Optional[str] = None  # Path to a directory which contains audio files
+    dataset_manifest: Optional[str] = None  # Path to dataset's JSON manifest
+
+    # General configs
+    output_filename: Optional[str] = None
+    batch_size: int = 32  # number of chunks to process in parallel
+    num_workers: int = 0  # unused
+    append_pred: bool = False  # Sets mode of work, if True it will add new field transcriptions.
+    pred_name_postfix: Optional[str] = None  # If you need to use another model name, rather than standard one.
+    random_seed: Optional[int] = None  # seed number going to be used in seed_everything()
+
+    # Set to True to output greedy timestamp information (only supported models)
+    compute_timestamps: bool = False
+
+    # Set to True to output language ID information
+    compute_langs: bool = False
+
+    # Chunked configs
+    chunk_len_in_secs: float = 30.0  # Chunk length in seconds
+    model_stride: int = 8  # Model downsampling factor, 8 for Citrinet and FasConformer models and 4 for Conformer models.
+
+    # Decoding strategy for MultitaskAED models
+    decoding: MultiTaskDecodingConfig = MultiTaskDecodingConfig()
+
+    # Set `cuda` to int to define CUDA device. If 'None', will look for CUDA
+    # device anyway, and do inference on CPU only if CUDA device is not found.
+    # If `cuda` is a negative number, inference will be on CPU only.
+    cuda: Optional[int] = None
+    amp: bool = False
+    audio_type: str = "wav"
+
+    # Recompute model transcription, even if the output folder exists with scores.
+    overwrite_transcripts: bool = True
+
+    # Config for word / character error rate calculation
+    calculate_wer: bool = True
+    clean_groundtruth_text: bool = False
+    langid: str = "en"  # specify this for convert_num_to_words step in groundtruth cleaning
+    use_cer: bool = False
+
+
+class MultiTaskAEDFrameBatchInfer(FrameBatchASR):
```

Comment:
This should be inside of nemos streaming_utils.py

Reply:
also may be add a deprecation banner if is only temp until nemo 2.0

Reply:
Done. I also changed default beam size to 1 for the multitask decoding